marienfressinaud/FreshRSS
# API implement OPML import/export

#fix https://github.com/FreshRSS/FreshRSS/issues/3421

```
/reader/api/0/subscription/export
/reader/api/0/subscription/import
```

How to test:

```sh
curl 'http://localhost:8080/api/greader.php/accounts/ClientLogin?Email=freshrss&Passwd=freshrss'

curl -v -H "Authorization:GoogleLogin auth=freshrss/6e54ad760d782b9647a2338129e93147e39ad26c" \
  'http://localhost:8080/api/greader.php/reader/api/0/subscription/export'

curl -v -H 'Authorization:GoogleLogin auth=freshrss/6e54ad760d782b9647a2338129e93147e39ad26c' \
  -H  'Content-Type: application/xml' -d '@/home/alexandre/Téléchargements/opml.xml' \
  'http://localhost:8080/api/greader.php/reader/api/0/subscription/import'
```

Check also that the CLI and normal exports/imports are still working.
